FileSystem.Fluent.Abstractions.Net5/Net5Directory.cs

```
@@ -0,0 +1,84 @@
+﻿using System;
+using System.IO;
+using FileSystem.Fluent.Abstractions.Core;
+
+namespace FileSystem.Fluent.Abstractions.Net5
+{
+    public class Net5Directory : IDirectory
+    {
+        public Net5Directory(string path)
+        {
+            Path = path;
+        }
+
+        public string Path { get; }
+
+        public IDirectory TryCreate()
+        {
+            try
+            {
+                Directory.CreateDirectory(Path);
+                return this;
+            }
+            catch (Exception e)
+            {
```

Let’s not write line, we should probably just throw the exception and let the clients handle it.